bin/videojs-generator-cleanup.js

```diff
@@ -34,51 +43,144 @@ var configs = {
       'scripts.watch:js',
       'scripts.watch:test'
     ]
+  },
+  v3: {
+    removals: [
+      '.babelrc',
+      'dist',
+      'docs/api',
+      'es5',
+      'jsdoc.json',
+      'scripts/banner.ejs',
+      'scripts/build-test.js',
+      'scripts/postversion.js',
+      'scripts/server.js',
+      'scripts/version.js',
+      'test/karma.conf.js',
+      'test/index.html',
+    ],
+    renames: {
+      'src/plugin.scss': 'src/css/index.scss',
+      'src/plugin.js': 'src/js/index.js',
+      'test/plugin.test.js': 'test/index.test.js'
+    },
+    pkg: [
+      'browserify',
+      'dependencies.browserify-versionify',
+      'dependencies.video.js',
+      'devDependencies.babel-cli',
+      'devDependencies.babel-plugin-transform-es3-member-expression-literals',
+      'devDependencies.babel-plugin-transform-es3-property-literals',
+      'devDependencies.babel-plugin-transform-object-assign',
+      'devDependencies.babel-preset-es2015',
```

Comment:
We shouldn't need the `-loose` preset. `loose` is available as an option on the es2015 preset. So, we just need to enable the option whenever we would use `es2015-loose`.